README.md

```diff
@@ -41,12 +41,17 @@ Use the scripts provided under `scripts` directory in order to run checks on eit
 
 ### Examples
 
-For running the example smart contracts found under `examples` you also need to install some further tools:
+For building the example smart contracts found under `examples` you will need to have `cargo-contract` installed.
 
-- `wabt` (WebAssembly Binary Toolkit)
-- `wasm-utils` (Parity's Wasm Utilities)
+```
+cargo install cargo-contract
+```
+
+Execute the following command in shell while in an example's directory:
```

Comment:
Between `cargo install cargo-contract` and `cargo contract build` we should maybe also include a section for `cargo contract new` since that is the logical step in between.

Reply:
While I understand and agree with you in general, I think this is not appropriate place for this kind of documentation: this particular piece is only about running examples.